docs/docs/external/getting-started/interop/asset-exchange/besu-besu.md

```diff
@@ -0,0 +1,141 @@
+---
+id: besu-besu
+title: "Asset Exchange: Besu with Besu"
+sidebar_label: Besu with Besu
+pagination_label: Besu with Besu
+pagination_prev: external/getting-started/interop/asset-exchange/overview
+---
+
+We divide this page into two sections, if you used default configuration in ledger initialization step, then go to section [AliceERC721 with BobERC20](#aliceerc721-with-boberc20), otherwise if you used hybrid tokens in `network`, then go to section [AliceERC1155 with BobERC20](#aliceerc1155-with-boberc20)
+
+| Notes |
+|:------|
+| The hash used in following steps can be replaced by any valid `SHA256` hash. |
+
+## AliceERC721 with BobERC20
+
+One Besu network transfers an non-fungible `AliceERC721` token with id `0` from Alice to Bob in exchange for a transfer of `10 BobERC20` tokens from Bob to Alice in the other network. We will use account `1` for Alice and account `2` for Bob in both networks.
+
+Run the following steps:
+
+1. Navigate to either the `samples/besu/besu-cli` folder in your clone of the Weaver repository.
+2. Run the following to verify the status of the assets owned by `alice` and `bob` in the two networks:
+   ```bash
+   ./bin/besu-cli asset get-balance --network=network1 --account=1
+   ./bin/besu-cli asset get-balance --network=network1 --account=2
+   ./bin/besu-cli asset get-balance --network=network2 --account=1
+   ./bin/besu-cli asset get-balance --network=network2 --account=2
+   ```
+3. Complete the asset exchange in either of the two different ways:
+  - Generate Secret-Hash Pair using following command (prints hash in base64):
```

Comment:
Since these are alternatives, mark the bullets as (a) and (b). Otherwise, the careless user may run them in sequence.